Fallout2/Fallout1in2/Mapper/source/scripts/GlobalScripts/gl_tellmeabout.ssl

```diff
@@ -148,153 +187,151 @@ procedure create_button begin
 procedure OpenInput begin
-   variable valid_source;
-
-   tma_data_array := load_array("tma_data_array");
-   valid_source := tma_data_array[0];
 
    // Only open interface if valid TMA data has been saved
    // (computer terminals and some other critters don't have TMAs)
-   if (valid_source > 0) then begin
-      fetch_input := 1;
+   if (tma_data_array[0] > 0) then begin
+      fetch_input := true;
       input := "";
       //debug("TMA input enabled!");
 
-      // Opening the interface clears the Reply window.
-      // TODO: Closing the interface should restore the Reply. Probably not possible right now.
-      dialog_message("");
+      // Reply window (clear the Reply window)
+      create_win("tellme_reply", (Scr_Width - 190), (Scr_Height - 64), 390, 64);
+      SelectWin("tellme_reply");
+      Display("PCX/tellmeabout_reply.pcx");
+      SetTextColor(0.0, 1.0, 0.0);
+      Format("What do you want to know?!", 8, 8, 200, 10, 0); // test reply
+      ShowWin;
 
       // Fake input / dialog option overlay
-      create_win("tellme_interface", (Scr_Width - 256), (Scr_Height + 24), 512, 160);
+      create_win("tellme_interface", (Scr_Width - 320), Scr_Height, 641, 190);
       SelectWin("tellme_interface");
-      draw_image("PCX/tellmeabout.frm", 0, 0, 0, 0);
+      FillWin(0.3, 0.3, 0.3);
+      sfall_func1("draw_image", "art\\intrface\\di_talk.frm");
+      draw_image("art\\intrface\\lsgbox.frm", 0, 320, 20, false);
 
       // CANCEL button
-      AddButton("button_cancel", 271, 100, 15, 16);
+      AddButton("button_cancel", 335, 120, 15, 16);
       AddButtonGFX("button_cancel", "PCX/lilreddn.pcx", "PCX/lilredup.pcx", "PCX/lilredup.pcx");
       AddButtonProc("button_cancel", do_on, do_off_small, do_down_small, do_up_cancel);
-
       // OK button
-      AddButton("button_ok", 145, 100, 15, 16);
+      AddButton("button_ok", 210, 120, 15, 16);
       AddButtonGFX("button_ok", "PCX/lilreddn.pcx", "PCX/lilredup.pcx", "PCX/lilredup.pcx");
       AddButtonProc("button_ok", do_on, do_off_small, do_down_small, do_up_ok);
 
-      ShowWin;
-
-      // Delete the input button
-      call delete_button;
-
-      // Create windows that will block the barter and dialog history buttons
-      create_win("barter_block", (Scr_Width + 266), (Scr_Height + 34), 25, 25);
-      SelectWin("barter_block");
-      draw_image("PCX/block_barter.frm", 0, 0, 0, 0);
-      ShowWin;
+      SetFont(103);
+      SetTextColor(0.52, 0.75, 0.15);
+      // TODO: Make getting text from game MSG-files
+      Format("Ask about what ?", 240, 70, 200, 30, 0);
+      Format("Done", 230, 118, 100, 30, 0);
+      Format("Cancel", 355, 118, 100, 30, 0);
 
-      create_win("history_block", (Scr_Width - 310), (Scr_Height + 153), 60, 30);
-      SelectWin("history_block");
-      draw_image("PCX/block_history.frm", 0, 0, 0, 0);
+      SetFont(100);
+      SetTextColor(0.0, 1.0, 0.0);
+      Format("$" + dude_caps, 4, 36, 70, 15, justifycenter);

```

Comment:
Choose the font here, try to SetFont (1)

Reply:
SetFont(1) looks much closer, but it's still not the same. Seems it is missing smooth edges. Also [this](https://i.imgur.com/ZCy4VSO.png) happens after closing the interface and moving the mouse over dialog options.

Reply:
try 2, 3

Reply:
or 101

Reply:
101 is the correct font. Just have to tweak the xy text coordinates now to make them fit with the original. Also seems like the gray window background is visible in a few places - you can see them [here](https://i.imgur.com/bRUK41D.png) pretty clearly if you quickly enable/disable the tma interface.

Reply:
Fixed the bottom right grey line with changing `create_win("tellme_interface", (Scr_Width - 320), Scr_Height, 640, 190);` << it was 641 before (on purpose?)

Reply:
Oh, and I found one more barter-button issue. :/ Your fix works when clicking the button on the interface, but if the player uses a dialog option to go into barter, the tma-button remains visible as before.